src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,46 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+    protected String by;
```

Comment:
The naming 'by' might be a little ambiguous. Perhaps 'timeBy' might be better since it is a time.